### ISSUE TITLE
ACE fails to open an epub file

### ISSUE THREAD

**Issue by jhswift** [daisy/ace-gui#175]
I have a file that appears to be a valid epub. Epubcheck reports No Problems and I can unpack and re-pack the file with eCanCrusher. 
But when I try to run it through ACE, the wheel spins for less than a second and stops showing the attached screen capture

![Ace-issue](https://github.com/daisy/ace-gui/assets/70376662/ca87fb41-edaa-4782-9b4a-3ed97742fa25)

The error message is:
ENOENT: no such file or directory, open 'C:\Users\jhswi\AppData\Local\Temp\tmp-3088-08YPku72vtwS\Karma et chaos\_unzipped_EPUB_\OEBPS\Karma%26chaos-Fr-ebook-v1.xhtml'

Is this a bug in ACE, or some kind of error in the file. 
I have also attached the zipped epub file.
Thanks
Jim Swift
[Karma et chaos.zip](https://github.com/user-attachments/files/16088942/Karma.et.chaos.zip)

**Comment by danielweck** [comment]
Thank you for reporting this. I see that some files inside the EPUB zip container have an ampersand character in their filenames, encoded (escaped) as `%26` inside XML attributes of the OPF package. This should indeed work, we need to fix how Ace handles escape sequences in filenames.

**Comment by jhswift** [comment]
Thanks, Daniel.
Will the 1.3.2 executable for the GUI also be updated?
Jim

**Comment by danielweck** [comment]
Yes, the imminent release of both Ace command line (aka CLI) and Ace App (aka GUI) will be version 1.3.3
There are a couple of blocking issues with the DAiSY Knowledge Base and its translations, but this should take long to sort out.

**Comment by jhswift** [comment]
Thanks, Daniel
Jim

**Comment by danielweck** [comment]
You can already try the automated builds at https://github.com/daisy/ace-gui/releases
(the installers are not code-signed, so the MacOS and Windows operating systems will give a warning message)

**Comment by jhswift** [comment]
Many thanks, Daniel.
I tested 1.3.3 with the files that had the two problems I reported.
No problems now.
Jim

**Comment by danielweck** [comment]
Thank you for getting back to me. I assume you tested on Windows?

**Comment by jhswift** [comment]
Yes, Daniel, on Windows 11.
Jim
